logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/featureConfig/IsAppLockEditableUseCaseTest.kt

```diff
@@ -0,0 +1,4 @@
+package com.wire.kalium.logic.feature.featureConfig
+
+class IsAppLockEditableUseCaseTest {
+}
```

Comment:
Please delete this file if  it is not possible to test this use case atm

Reply:
Oh wow, this was not supposed to be here 🤔